static/js/mobile_redirect.js

```diff
@@ -0,0 +1,25 @@
+var mobile_redirect = (function () {
+  var exports = {};
+
+  function hide_app_alert() {
+      $('#alert-bar-container').slideUp(100);
+  }
+
+  function show_app_alert(contents) {
+    $('#custom-alert-bar-content').html(contents);
+    $('#alert-bar-container').show();
+    $('#alert-bar-container .close-alert-icon').expectOne().click(hide_app_alert);
+  }
+
+  exports.try_to_display = function () {
+    var alert_contents = "<i class='icon-vector-mobile'></i>What's better than " + page_params.product_name + " in your mobile browser? The <a href='/accounts/login/mobile_redirect/' target='_blank'>"+ page_params.product_name + " Mobile app</a>!";
```

Comment:
Can you switch to generating this message in the templates?  

Zulip is in the process of being internationalized, so we'd like to avoid having raw strings constructed in javascript.